src/SAWScript/JavaBuiltins.hs

```diff
@@ -327,7 +329,7 @@ mkMixedExpr t = do
   sc <- lift getSharedContext
   let exts = getAllExts t
       extNames = map ecName exts
-  jes <- mapM (getJavaExpr "mkMixedExpr") extNames
+  jes <- mapM (getJavaExpr "mkMixedExpr" . Text.unpack . toShortName) extNames -- TODO?????
```

Comment:
what's the TODO?

Reply:
The TODO here is that using `toShortName` for anything other than user output is very suspicious.  This line is no worse than what was there before (I think), but someone who understands how the Java stuff is put together should think about what happens if there are name conflicts and such.

Reply:
We shouldn't worry about this code too much; it's only used by the deprecated jss-based java verification stuff. We will be removing it before too long.